tests/test_flagController.py

```diff
@@ -1,7 +1,10 @@
 import pytest
-from unittest.mock import patch, MagicMock
-from twinTrim.flagController import handleAllFlag
-from twinTrim.dataStructures.fileFilter import FileFilter
+import os
+from pathlib import Path
+from unittest.mock import patch, MagicMock, mock_open
+from ..twinTrim.flagController import handleAllFlag
```

Comment:
This is where the issue is.
you were doing this

from ..twinTrim.flagController import handleAllFlag
from ..twinTrim.dataStructures.fileFilter import FileFilter
from ..twinTrim.flagController import find_duplicates


you should have done this 

from twinTrim.flagController import handleAllFlag
from twinTrim.dataStructures.fileFilter import FileFilter
from twinTrim.flagController import find_duplicates

Pointed out by @Kota-Karthik